ts/sagas/wallet.ts

```diff
@@ -24,10 +90,203 @@ function* fetchTransactions(
 }
 
 function* fetchCreditCards(
-  loadCards: () => Promise<ReadonlyArray<CreditCard>>
+  loadCards: () => Promise<ReadonlyArray<Wallet>>
+): Iterator<Effect> {
+  const cards: ReadonlyArray<Wallet> = yield call(loadCards);
```

Comment:
this call can fail (e.g. timeout), you should handle that

Reply:
The retry logic will be added when the data will actually be fetched from the API (i.e. in a different PR) (btw -- you've already worked on adding this kind of logic, right?)

Reply:
Has the expected behavior already been defined for this case as far as you know?

Either way, I would say that this behavior still belongs to the PR that takes care of fetching the cards data/transactions data (which need some updating anyway b/c of the news that came from pagoPA last week)

Reply:
I think so, we have some error screens now in Invision, I guess the default behavior is to show an error screen and ask the user if she wants to retry or abort. - @spizzichi @matteodesanti